### DISCUSSION TITLE
An option to delete playlists from the library menu

### DISCUSSION THREAD

**Discussion by Spyder-6** [gokadzev/Musify#445] [Ideas]
as i said in the title, an  button to delete specific user custom playlist helps for a lot of guys like me as my older playlist dosen't work

**Comment by gokadzev** [comment]
@Spyder-6 I mentioned in issue #439 that you should back up your Musify data and send it to help identify the exact problem, but I haven't received a response. Regarding this feature request, it's already possible to delete a playlist by holding down on the playlist you want to delete (long press for several seconds)

**Reply by Spyder-6** [reply]
how do i send it

**Reply by gokadzev** [reply]
> how do i send it

attach it here, as soon as I download, I'll remove your comment and that's it.
